### PR TITLE
feat: block accounts

### PR DESCRIPTION
**Pull request by cartcrom**
for ongoing incident

### CODE REVIEW

**Review by AndrewMohawk — APPROVED** [review]
Temp approval 👍

**Review by cbachmeier — APPROVED** [review]

File: src/hooks/manual_block_list.ts [hunk 1]
```diff
@@ -0,0 +1,1197 @@
...
+  '0x2cc061427ad01116d2d57d3a233efd326c0175ff': {},
+  '0x4447bcbe4be076c1a188a0178c5137da218698cd': {},
+  '0x8b23eebeefb34537c94a14430c564b3827275823': {},
+  '0xfdcaf6fed2e91a2f7792d488820de7fdc590a6d5': {},
+  '0x4aa329613c50d5ee76e8c3bb55185dc357e4978f': {},
+  '0x8abd9c7efa840907a44f633dd91ce4137600308b': {},
+  '0x97fe4ed781ba0a07fbd9323109d0458478aced9e': {},
+  '0xed7f81c87bc15b4be85e26cb49560528838dbd58': {},
+  '0x4deeb19cc5be551e3e7ca1d54e7960542ceed642': {},
+  '0xbe437804b82995c0c9ed63e59fe2d090f312f2b6': {},
```
Comment (tinaszheng) [inline src/hooks/manual_block_list.ts:1196]: Why not just make this a set?

**Review by cbachmeier — APPROVED** [review]

File: src/hooks/manual_block_list.ts (hunk 1 above)
Comment (cartcrom) [reply src/hooks/manual_block_list.ts:1196]: cause faster to paste in what i have

Comment (cartcrom) [reply src/hooks/manual_block_list.ts:1196]: same end result so working faster not smarter rn

**Review by mattbspector — APPROVED** [review]

**Pull request closed without merging**